lua/curl/parser.lua

```diff
@@ -68,9 +68,12 @@ local format_command_for_curl = function(lines)
 		if found_first_json_char(json_nesting_stack, cleaned_line) then
 			opening_json_char = cleaned_line:sub(1, 1)
 			closing_json_char = opening_json_char == "[" and "]" or "}"
-			is_line_json_close(json_nesting_stack, opening_json_char, closing_json_char, cleaned_line)
-
 			cleaned_line = "'" .. cleaned_line
+
+			-- if the all json content is in one line
```

Comment:
Dont need this comment, this if check will work for all cases

Reply:
That makes sense. It's been taken care of.